rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -0,0 +1,104 @@
+:_content-type: REFERENCE
+[id="rosa-release-notes"]
+= {product-title} release notes
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-release-notes
+
+toc::[]
+
+{product-title} ROSA is a fully-managed, turnkey application platform that allows you to focus on delivering value to your customers by building and deploying applications. Red Hat and AWS Site reliability engineering (SRE) experts manage the underlying platform so you do not have to worry about the complexity of infrastructure management. ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to further accelerate the building and delivering of differentiating experiences to your customers.
+
+[id="about-this-release-section_{context}"]
+== About this release
+
+{product-title} (ROSA) https://access.redhat.com/errata/RHEA-2022:2252[RHEA-2022:2252] is now available. This release uses Kubernetes 1.23 with CRI-O runtime. New features, changes, and known issues that pertain to ROSA 4.10 are included in this topic.
+
+ROSA clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for ROSA, you can deploy OpenShift clusters to either on-premise or cloud environments.
+
+[id="about-this-release_{context}"]
+== ROSA layered and dependent component support and compatibility
+The scope of support for layered and dependent components of ROSA changes independently of the ROSA version. To determine the current support status and compatibility for an add-on, refer to its release notes. For more information, see the xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle].
+
+[id="rosa-new-features"]
+== New features and enhancements
+
+[id="rosa-product-enhancements"]
+=== Product enhancements
+
+The following list describes the key features of ROSA:
+
+* **ROSA UI wizard**: The new ROSA provisioning wizard UI is now available as an AWS STS-only experience. You can access the new UI wizard at https://console.redhat.com/openshift/create/rosa/wizard.
+
+* **Latest Command Line Interface and New Operator**: ROSA Command Line Interface (CLI) 1.1.11 released on March 9, 2022 and later enabled the creation of ROSA 4.10 clusters. Please ensure you are always using the latest ROSA CLI for the best experience. OpenShift 4.10 has a new operator and cloud network configuration controller. Therefore, a new operator role and policy will be added when you upgrade to OpenShift 4.10, or it will be a part of new cluster deployments. During an upgrade to OpenShift 4.10, you will be prompted by the ROSA CLI to add a new role and policy.
```

Comment:
- Latest Command Line Interface and New Operator -> should be sentence case, i.e. "Latest command-line interface and new Operator", you could even just say "CLI" here; "Latest CLI and new Operator"

- In the line after, "ROSA Command Line Interface (CLI)" I think it's OK to just use "CLI" - I'm pretty sure that's one of those terms that is used so often that we don't spell it out this way.

- Suggestion: "ROSA CLI 1.1.11 was released on March 9, 2022 and enabled the creation of ROSA 4.10 clusters."

- " OpenShift 4.10 has a new operator" -> use OpenShift attribute where possible for all mentions (per earlier comment. Operator should always be capitalized as far as I know. I see a few more instances of "OpenShift" and "operator" in this PR but I won't comment on all of them, please address this for any instances where it's relevant though.

Reply:
Incorporated this feedback.